src/EnergyPlus/DXCoils.cc

```diff
@@ -7048,8 +7048,10 @@ namespace DXCoils {
 		NodeData & WaterOutletNode = Node( Coil.WaterOutNode );
 
 		// If heat pump water heater is OFF, set outlet to inlet and RETURN
+		// Also set the heating energy rate to zero
 		if ( PartLoadRatio == 0.0 ) {
 			WaterOutletNode = WaterInletNode;
+			Coil.TotalHeatingEnergyRate = 0.0;
```

Comment:
The tank is getting the heat flow from the coil via this variable now (as of earlier this year). Whenever the PLR was zero, it was using the heat rate from the previous call instead of zero as it should.